src/backend/distributed/planner/multi_explain.c

```diff
@@ -338,6 +347,18 @@ ExplainJob(Job *job, ExplainState *es)
 	ExplainOpenGroup("Job", "Job", true, es);
 
 	ExplainPropertyInteger("Task Count", NULL, taskCount, es);
+	if (es->analyze)
+	{
+		Task *task = NULL;
+		uint64 totalReceivedDataForAllTasks = 0;
+		foreach_ptr(task, taskList)
+		{
+			totalReceivedDataForAllTasks += task->totalReceivedData;
```

Comment:
Do we need to take into account placement count for DML with RETURNING?

Reply:
I tried it out and yes we did.